baseball/src/App.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import Dashboard from "./Components/Dashboard";
+import Display from "./Components/Display";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      ballCount: 0,
+      strikeCount: 0
+    };
+  }
+
+  strikeHandler = event => {
+    if (this.state.strikeCount === 3) {
+      this.setState({ strikeCount: 0, ballCount: 0 });
+    } else {
+      this.setState({ strikeCount: this.state.strikeCount + 1 });
+    }
+  };
+
+  ballHandler = event => {
+    event.preventDefault();
+
+    if (this.state.ballCount === 4) {
+      this.setState({ strikeCount: 0, ballCount: 0 });
+    } else {
+      this.setState({ ballCount: this.state.ballCount + 1 });
+    }
+  };
+
+  hitHandler = event => {
+    this.setState({ strikeCount: 0, ballCount: 0 });
+  };
+
+  foulHandler = event => {
+    if (this.state.strikeCount == 0) {
+      this.setState({ strikeCount: 1 });
+    } else if (this.state.strikeCount == 1) {
+      this.setState({ strikeCount: 2 });
```

Comment:
These two conditions seem repetitive since their setStates are doing the same thing (incrementing strikeCount by 1); do you think you could get rid of the extra check? For example:
```javascript
let { strikeCount } = this.state;
if (strikeCount == 0 || strikeCount == 1) {
  strikeCount++;
  this.setState({ strikeCount });
}
```